const Exception = (message: string) => {
    return {
        message,
    };
};
export default Exception;
